recipes/wally/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set version = "0.2.3" %}
+{% set revision = "93442348984044ea12608c323b7480487ab70eba" %}
```

Comment:
Can you tag a release rather than specifying an exact commit? We can automatically create PRs to update things when there's a new release, but that doesn't work when recipes mention exact commits.

Reply:
This commit is a release [v0.2.3](https://github.com/tobiasrausch/wally/releases/tag/v0.2.3) but I need the submodules.